docs/dev/Lua API.rst

```diff
@@ -5012,6 +5012,40 @@ The widget implements:
 
   Same as with an ordinary list.
 
+TabBar class
+------------
+
+This widget implements a set of one or more tabs to allow navigation between groups of content. Tabs automatically wrap on
+the width of the window and will continue rendering on the next line(s) if all tabs cannot fit on a single line.
+
+:labels: A table of strings; entry representing the label text for a single tab. The order of the entries
+         determines the order the tabs will appear in.
+:get_cur_page: Returns the current "page". This function does not have a default implementation; you must provide
+               an implementation that returns the current value of whichever variable your script uses to keep track of the
+               current "page" (this does not need to relate to an actual Pages widget).
```

Comment:
shouldn't this be a little more prescriptive? shouldn't it return the index that corresponds to the active tab? otherwise the default `get_pens` function won't work.

Reply:
Yeah, that's pretty much what I was trying to get across here. I struggle with doc writing a lot haha. I'll try to reword it.

Reply:
Attempted to make it a bit more prescriptive.